ts/components/PinCreationForm.tsx

```diff
@@ -40,14 +46,16 @@ export const PinCreationForm = ({ onSubmit }: Props) => {
   const [isPinConfirmationDirty, setIsPinConfirmationDirty] =
     React.useState(false);
 
-  const isPinValid = !isPinDirty || pin.length === pinLength;
+  const isPinValid =
+    !isPinDirty ||
+    isValidPinNumber(pin) ||
+    (isDevEnv && pin.length === pinLength);
```

Comment:
I wan't disable it in development... We could use 010101 or 567890. Anyway is a good way to preserve the policy also in development mode. IMHO.